acceptance_testing.go

```diff
@@ -1124,5 +1123,60 @@ func (a acceptanceTest) context(t *testing.T) context.Context {
 	return ctx
 }
 
+func (a acceptanceTest) configureSource(ctx context.Context, t *testing.T, src Source) error {
+	return configureSource(
+		ctx,
+		src,
+		a.driver.SourceConfig(t),
+		a.driver.Connector().NewSpecification().SourceParams,
+	)
+}
+
+func (a acceptanceTest) configureDestination(ctx context.Context, t *testing.T, dest Destination) error {
+	return configureDestination(
```

Comment:
I feel like this either needs to be a method or a function, but why having both?

Reply:
It's just to make it a bit simpler to configure a destination that's in an `acceptanceTest`, because an `acceptanceTest` can fetch the configuration and the parameters itself.